src/Files.App/UserControls/Widgets/RecentFilesWidget.xaml.cs

```diff
@@ -109,27 +111,43 @@ public RecentFilesWidget()
 
 		private void ListView_RightTapped(object sender, RightTappedRoutedEventArgs e)
 		{
-			ItemContextMenuFlyout = new CommandBarFlyout { Placement = FlyoutPlacementMode.Full };
-			ItemContextMenuFlyout.Opening += (sender, e) => App.LastOpenedFlyout = sender as CommandBarFlyout;
-			if (e.OriginalSource is not FrameworkElement element || element.DataContext is not RecentItem item)
+			// Ensure values are not null
+			if (e.OriginalSource is not FrameworkElement element ||
+				element.DataContext is not RecentItem item)
 				return;
 
-			var menuItems = GetItemMenuItems(item, false);
+			// Create a new Flyout
+			var itemContextMenuFlyout = new CommandBarFlyout()
+			{
+				Placement = FlyoutPlacementMode.Full
+			};
+
+			// Hook events
+			itemContextMenuFlyout.Opening += (sender, e) => App.LastOpenedFlyout = sender as CommandBarFlyout;
+			itemContextMenuFlyout.Opened += (sender, e) => OnRightClickedItemChanged(null, null);
```

Comment:
@0x5bfa #14379 can be fixed by commenting out this code. Is this code necessary?